cartography/graph/job.py

```diff
@@ -74,12 +74,15 @@ def as_dict(self) -> Dict:
         }
 
     @classmethod
-    def from_json(cls, blob: str, short_name: str = None):
+    def from_json(cls, blob: str, short_name: str = None, parameters: Dict = None,):
         """
         Create a job from a JSON blob.
         """
         data: Dict = json.loads(blob)
         statements = _get_statements_from_json(data, short_name)
+        if parameters.get('service_label', None):
```

Comment:
can we avoid changing graph/job.py? is it possible to implement this outside this file.

Reply:
no because if we pass label name directly in cleanup job query then it takes as string and gives error

Reply:
there is no anther way to use this logic